### PR TITLE
Intro to Bitwise Operators Quiz

### PR DESCRIPTION
**Pull request by alexyd88**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [x] I have linked this PR to any issues that it closes.

### CODE REVIEW

File: content/3_Silver/Intro_Bitwise.mdx [hunk 1]
```diff
@@ -342,3 +342,109 @@ def prod(a: int, b: int) -> int:
...
+  <Quiz.Question>
+    Which of the following will set the kth bit of int x as true? Assume you do not know what the value of the kth bit currently is.
+    <Quiz.Answer>
+      $x \&= (1<<k);$
+      <Quiz.Explanation>
+        Incorrect. This will set the number to $1<<k$ if the kth bit was true, and 0 if the kth bit was false.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      $x |= (1<<k);$
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Bitwise.mdx:357]: is this meant to be in latex

File: content/3_Silver/Intro_Bitwise.mdx [hunk 2]
```diff
@@ -342,3 +342,109 @@ def prod(a: int, b: int) -> int:
...
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      __builtin_ctz(x)
+      <Quiz.Explanation>
+        Incorrect. This gives the number of zeros at the end of the number.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      __builtin_popcount(x)
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Bitwise.mdx:384]: maybe avoid c++ specific stuff
or put it in a languagesection

File: content/3_Silver/Intro_Bitwise.mdx (hunk 1 above)
Comment (SansPapyrus683) [reply content/3_Silver/Intro_Bitwise.mdx:357]: like all of these seem to be better in source code format

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
nice!

File: content/3_Silver/Intro_Bitwise.mdx [hunk 3]
```diff
@@ -342,3 +342,82 @@ def prod(a: int, b: int) -> int:
 </LanguageSection>
 
 <Problems problems="probs" />
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    Which of the following will set the kth bit of int x as true? Assume you do not know what the value of the kth bit currently is.
+    <Quiz.Answer>
+      x \&= (1 \<\< k)
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Bitwise.mdx:351]: wait what happened to the latex

Comment (SansPapyrus683) [reply content/3_Silver/Intro_Bitwise.mdx:351]: i am the dumbest man alive

**Review by SansPapyrus683 — APPROVED** [review]

File: content/3_Silver/Intro_Bitwise.mdx [hunk 4]
```diff
@@ -342,3 +342,82 @@ def prod(a: int, b: int) -> int:
...
+      `x += (1 \<\< k)`
+      <Quiz.Explanation>
+        Incorrect. If the kth bit is already true, it will make the kth bit false and mess up the rest of x.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which gives the number of subsets for an array of size n? (Include the empty subset)
+    <Quiz.Answer>
+      `n \<\< 1`
```
Comment (SansPapyrus683) [inline content/3_Silver/Intro_Bitwise.mdx:372]: just asking, are the backslashes necessary

Comment (alexyd88) [reply content/3_Silver/Intro_Bitwise.mdx:372]: yeah, otherwise it gives error since it thinks '<' is the start of a new section

**Review by ryanchou-dev — APPROVED** [review]
maybe add another problem? (optional)

**Comment by SansPapyrus683** [comment]
eh, we can add it later

**Pull request merged**